source/macos/AdaptiveCards/AdaptiveCards/Views/ACRCollectionView.swift

```diff
@@ -69,23 +69,45 @@ class ACRCollectionView: NSCollectionView {
 
 // MARK: DataSource for CollectionView
 class ACRCollectionViewDatasource: NSObject, NSCollectionViewDataSource {
+    var imageViews: [ImageSetImageView] = []
+    var images: [ACSImage] = []
+    var imageSize: ACSImageSize = .medium
+    
+    func registerImageViews(acsImages: [ACSImage], rootView: ACRView, size: ACSImageSize) {
+        self.images = acsImages
+        self.imageSize = size
+        for image in images {
+            let imageView = ImageSetImageView()
+            guard let url = image.getUrl() else { continue }
+            rootView.registerImageHandlingView(imageView, for: url)
+            imageViews.append(imageView)
+        }
+    }
```

Comment:
```suggestion
    let imageViews: [ImageSetImageView]
    let images: [ACSImage]
    let imageSize: ACSImageSize
    
    init(acsImages: [ACSImage], rootView: ACRView, size: ACSImageSize) {
        self.images = acsImages
        self.imageSize = size
        for image in images {
            let imageView = ImageSetImageView()
            guard let url = image.getUrl() else { continue }
            rootView.registerImageHandlingView(imageView, for: url)
            imageViews.append(imageView)
        }
    }
```
I guess this can be wrapped inside `init` method.